resources/views/informer.blade.php

```diff
@@ -0,0 +1,5 @@
+@if(session('honeybadger_last_error'))
+    <div class="{{ $class }}">
+        <small>{{ $text }} {{ session('honeybadger_last_error') }}</small>
```

Comment:
Would it be possible to make the text small using a CSS class instead of the tag? It's not possible to override the small style via the CSS option, which would be a nice feature (basically the div is just a container for the text and the user can style it however they want).

Reply:
Done.